### PR TITLE
Adds tests for feed-linker and feed-footer component

### PR DESCRIPTION
**Pull request by shreyanshdwivedi**
**Changes proposed in this pull request**
- Adds tests for feed-linker and feed-footer component

**Screenshots (if appropriate)** 

**Link to live demo: **  http://pr-946-fossasia-loklaksearch.surge.sh

**Related to #414**

### CODE REVIEW

**Comment by shreyanshdwivedi** [comment]
@simsausaurabh @praveenojha33 please review

**Review by simsausaurabh — APPROVED** [review]

**Review by sakshee-19 — APPROVED** [review]

**Comment by shreyanshdwivedi** [comment]
@praveenojha33 please review

**Review by praveenojha33 — APPROVED** [review]

**Comment by shreyanshdwivedi** [comment]
@simsausaurabh please merge

File: src/app/feed/feed-linker/feed-linker.component.spec.ts [hunk 1]
```diff
@@ -1,9 +1,55 @@
...
+		expect(component.shardArray.length).toBe(4);
+	}));
+
+	it('should call generateShards() when useAll is false', async(() => {
+		const fixture = TestBed.createComponent(FeedLinkerComponent);
+		const component = fixture.debugElement.componentInstance;
+
+		component.useAll = false;
+		component.hashtags = [
+			'fossasia',
```
Comment (simsausaurabh) [inline src/app/feed/feed-linker/feed-linker.component.spec.ts]: hashtags to be given should be: ```FOSSASIA```. We follow this in our tweets too.

**Review by simsausaurabh — CHANGES_REQUESTED** [review]
Make one change.

**Comment by shreyanshdwivedi** [comment]
@simsausaurabh please review

**Review by simsausaurabh — APPROVED** [review]

**Comment by simsausaurabh** [comment]
@shreyanshdwivedi For your future PRs for increasing test coverage try only to add something meaningful to test since you have got familiar with writing test cases for the current codebase.

**Pull request merged**